backend/src/manager/mod.rs

```diff
@@ -569,22 +664,31 @@ impl PersistantContainer {
     async fn done_waiting(&self) {
         self.wait_for_start.0.send(false).unwrap();
     }
+
+    fn exec_command(&self) -> ExecCommand {
```

Comment:
:notebook: This was once a trait, couldn't get the trait working so went with this. Easier ish.